exporter/opentelemetry-exporter-otlp-proto-grpc/tests/logs/test_otlp_logs_exporter.py

```diff
@@ -212,6 +214,10 @@ def test_exporting(self):
             OTEL_EXPORTER_OTLP_LOGS_ENDPOINT: "logs:4317",
             OTEL_EXPORTER_OTLP_LOGS_CERTIFICATE: THIS_DIR
             + "/../fixtures/test.cert",
+            OTEL_EXPORTER_OTLP_LOGS_CLIENT_CERTIFICATE: THIS_DIR
```

Comment:
Are we handling and testing the previous functionality, where client keys/certs are not used?

Reply:
Added tests for 3, tyvm :)